### PR TITLE
Add initial Travis-CI configuration

### PR DESCRIPTION
**Pull request by eustas**
See #126

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+      dist: trusty
+      sudo: required
+      env: BUILD_SYSTEM=bazel
+      addons:
+        apt:
+          sources:
+            - ubuntu-toolchain-r-test
+          packages:
+            - wget
+            - pkg-config
```
Comment (robryk) [inline .travis.yml]: If this is for compiling with Bazel, we don't need pkg-config

File: .travis.sh [hunk 2]
```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+case "$1" in
+    "install")
+	case "${TRAVIS_OS_NAME}" in
+	    "linux")
+		wget https://github.com/bazelbuild/bazel/releases/download/0.4.5/bazel_0.4.5-linux-x86_64.deb
+		echo 'b494d0a413e4703b6cd5312403bea4d92246d6425b3be68c9bfbeb8cc4db8a55  bazel_0.4.5-linux-x86_64.deb' | sha256sum -c --strict || exit 1
+		sudo dpkg -i bazel_0.4.5-linux-x86_64.deb
```
Comment (robryk) [inline .travis.sh]: Just for my curiosity: any reason to do it differently than https://github.com/bazelbuild/bazel/blob/master/site/versions/master/docs/install-ubuntu.md#1-add-bazel-distribution-uri-as-a-package-source-one-time-setup ?

Comment (eustas) [inline .travis.sh]: No particular reason. Just used a snippet that works.
Will try "official" recipe and see if it runs well.

File: .travis.yml (hunk 1 above)
Comment (eustas) [inline .travis.yml]: Looks like a legacy from times when bazel was to be build before use. Will remove.

File: .travis.sh (hunk 2 above)
Comment (eustas) [inline .travis.sh]: No github projects use "official" way, but about 250+ projects download package directly.
Going to leave as-is for now.

**Pull request merged**